std/typecons.d

```diff
@@ -6544,6 +6559,23 @@ pure @system unittest
     RefCounted!(int*) c;
 }
 
+@system unittest // Issue 20502
```

Comment:
Is there a reason for this to be `@system` other than the cast?

Reply:
RefCounted in  general is `@system`.